rebac-admin-backend/v1/response.go

```diff
@@ -26,6 +26,15 @@ func writeErrorResponse(w http.ResponseWriter, err error) {
 
 // mapErrorResponse returns a Response instance filled with the given error.
 func mapErrorResponse(err error) *resources.Response {
+	// Theoretically, this should never happen, but we anyway have to check for
+	// a nil argument.
+	if err == nil {
+		return &resources.Response{
+			Status:  http.StatusOK,
+			Message: "",
+		}
+	}
+
```

Comment:
nitpick(non-blocking): even though it doesn't hurt to check, being `mapErrorResponse` an unexported function we should not worry about err being nil

Reply:
Yeah, that's why I put the comment above it. A `nil` argument should never happen, but although this is an internal method, nevertheless, we need to do either of the following to fool-proof it: 
- Have tests to assert that `mapErrorResponse` is *never* called with a `nil` argument, which is hard to actually test.
- Accept that it's possible (though, very unlikely) and handle it like I did and have a test to verify this is the intended behavior.